src/system/ExtensionsModule/Api/ApiInterface/CapabilityApiInterface.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Copyright Zikula Foundation 2016 - Zikula Application Framework
+ *
+ * This work is contributed to the Zikula Foundation under one or more
+ * Contributor Agreements and licensed to You under the following license:
+ *
+ * @license GNU/LGPLv3 (or at your option, any later version).
+ *
+ * Please see the NOTICE file distributed with this source code for further
+ * information regarding copyright and licensing.
+ */
+
+namespace Zikula\ExtensionsModule\Api\ApiInterface;
+
+use Zikula\ExtensionsModule\Entity\ExtensionEntity;
+
+/**
+ * Interface CapabilityInterface
+ * Constant strings for use in the Core.
+ * in composer.json extra/zikula/capabilities definition, use the actual string.
+ * @package Zikula\ExtensionsModule
```

Comment:
`@package` is old and not needed - you should tell PHPStorm to not include it in generated doc blocks.